numpy/_core/include/numpy/ufuncobject.h

```diff
@@ -65,6 +65,9 @@ typedef int (PyUFunc_TypeResolutionFunc)(
                                 PyObject *type_tup,
                                 PyArray_Descr **out_dtypes);
 
+typedef int (PyUFunc_ProcessCoreDimsFunc)(
```

Comment:
Need to add a bit of docs here.  Also need to add the same/similar docs for to the proper ufunc documentation.

Reply:
Yes, code docs and user docs both need work.